imod_coupler/drivers/ribametamod/ribametamod.py

```diff
@@ -303,17 +310,18 @@ def update(self) -> None:
 
         self.mf6.prepare_time_step(0.0)
         self.delt_gw = self.mf6.get_time_step()
-        self.subtimesteps_sw = range(1, int(self.delt_gw / self.delt_sw) + 1)
 
         if self.has_ribasim:
+            self.ribasim.update_subgrid_level()
```

Comment:
Move this into a separate method called `update_ribasim_modflow()` or `exchange_ribasim_modflow()`.